### PR TITLE
WIP: Update to store a range from 10-100 for age

### PR DESCRIPTION
**Pull request by gnarf**
This converts the age to a bunch of options :)

Bonus points: The frontend still handles this change (though the ui is ugly)

### CODE REVIEW

**Comment by boazsender** [comment]
+1, this works for me. 

I think we should land a new range front end with this before merging to master..

**Comment by gnarf** [comment]
Added some tweaking to the front end to get us started with showing range...

We probably want to figure out a better UX / style for the front end components... I'm going to have to take off, so if anyone else wants to add commits to this branch with UX / UI improvements, feel free!

**Comment by boazsender** [comment]
+1, this looks good. @isaacdurazo are you able to style this?

File: backend/db/migrations/j12bouy3.age-range.sql [hunk 1]
```diff
@@ -0,0 +1,50 @@
+ALTER TABLE annotation_attribute
+  ADD type TEXT;
+
+UPDATE annotation_attribute
+  SET type = 'select-one'
+  WHERE NOT(name = 'Perceived Age');
```
Comment (tkellen) [inline backend/db/migrations/j12bouy3.age-range.sql]: Just calling out that you can do `WHERE name != 'Perceived Age'` No need to change it, I just find that syntax a little more readable.

**Comment by tkellen** [comment]
looking good to me!

**Comment by boazsender** [comment]
I'm landing this and opening a new issue to iterate on the slider ux.

**Pull request merged**